utils/build-script-impl

```diff
@@ -1556,6 +1556,8 @@ for host in "${ALL_HOSTS[@]}"; do
                           not
                       )
                     fi
+                else
+                    build_targets=(clean)
```

Comment:
Do you know why clean is here? (I don't know either). Can you figure that out and document it?